include/circt/Dialect/SV/SVStatements.td

```diff
@@ -657,3 +658,15 @@ def InfoOp : NonfatalMessageOp<"info"> {
     This system task indicates a message with no specific severity.
   }] # messageDescription;
 }
+
+def DepositOp : SVOp<"cadence.deposit", 
+    [ProceduralOp, VendorExtension, InOutTypeConstraint<"src", "dest">]> {
+  let summary = "`$deposit` system task";
+  let description = [{
+    This system task sets the value of a net or variable, but doesn't hold it.
+    This is a Cadence vendor extension.
+  }];
+  let arguments = (ins InOutType:$dest, InOutElementType:$src);
+  let results = (outs);
+  let assemblyFormat = "$dest `,` $src  attr-dict `:` qualified(type($src))";
+}
```

Comment:
It looks like the big 3 all support this:
  - [VCS Manual](http://cc.ee.ntu.edu.tw/~ric/teaching/SoC_Verification/S06/Homework/HW1/vcs.pdf) shows it
  - [Modelsim Manual](https://cseweb.ucsd.edu/classes/fa10/cse140L/lab/docs/modelsim_user.pdf) shows it

Verilator does not appear to support this, though you could hack up deposit via a C++ testbench working with Verilator.

My point being I don't know if this should be `nonstandard.deposit` or something like that. I also can't tell who implemented it first. Also, the comment should be updated to state that Synopsys, VCS, and Mentor support this.